test/conformance/kubetestgen/main.go

```diff
@@ -0,0 +1,211 @@
+package main
```

Comment:
add copyright headers to main.go and types.go

Reply:
main.go should be named kubetestgen.go

Reply:
Add bazel BUILD file

Reply:
> Add bazel BUILD file

It looks like bazel does not support modifying the state of the workspace by [design](https://stackoverflow.com/questions/47764221/how-to-write-files-to-current-directory-instead-of-bazel-out). Thus, when we run the script via `bazel run` it won't be able to write to the workspace and will only be able to modify files in the generated output directories.

I was able to get the script working via using only `bazel build ...` and manually running the script in the `bazel-out` symlinked directory.

Full command invoked from the root directory:

```
bazel build //test/conformance/kubetestgen:kubetestgen
./bazel-out/k8-fastbuild/bin/test/conformance/kubetestgen/linux_amd64_stripped/kubetestgen --resource PodSpec --area pod --schema test/conformance/kubetestgen/schema.json --dir test/conformance/behaviors/gene
rated/
```

The location of the generated binary is platform dependent, so we're not able to generalize this into a Makefile. These problems do not exist when we use `go build ...`